Tests/SessionManagerTests.swift

```diff
@@ -131,7 +177,7 @@ class SessionManagerTestCase: BaseTestCase {
         XCTAssertNil(manager, "manager should be nil")
     }
 
-    // MARK: Start Requests Immediately Tests
+    // MARK: Tests - Start Requests Immediately
```

Comment:
These are clearly all tests, so perhaps it's not the ordering that matters but the existence of the word at all? I don't care either way.